src/realm/util/interprocess_condvar.cpp

```diff
@@ -190,8 +241,10 @@ void InterprocessCondVar::init_shared_part(SharedPart& shared_part)
 
 void InterprocessCondVar::release_shared_part()
 {
-#ifdef REALM_CONDVAR_EMULATION
+#ifdef REALM_CONDVAR_EMULATION 
+#ifndef _WIN32
     File::try_remove(m_resource_path);
+#endif
 #else
 #endif
```

Comment:
As I wrote, _all_ platforms should be covered in the cases (even if empty), otherwise this _adds_ the risk that the `#else` part is erroneously and silently used, when it was not intended.